hurumap/settings.py

```diff
@@ -103,30 +99,65 @@
 HURUMAP['map_zoom'] = 6
 
 
+HURUMAP['geodata'] = 'hurumap.geo.GeoData'
+HURUMAP['mapit'] = {}
+HURUMAP['geometry_data'] = {
+    '2009': {
+        'country': 'geo/country.topojson',
+        'county': 'geo/county.topojson'
+
+    }
+}
+
+HURUMAP['USE_MAPIT'] = os.environ.get('USE_MAPIT', False)
```

Comment:
I think it would have been better to cast it using [`strtobool`](https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool)

Keeping it as string forces us to use `if HURUMAP['USE_MAPIT'] == "True"` **everywhere**.